unison-src/transcripts-round-trip/main.output.md

```diff
@@ -928,9 +936,10 @@ broken = cases
     broken : Optional Nat -> ()
     broken = cases
       Some
-        loooooooooooooooooooooooooooooooooooooooooooooooooooooooong | loooooooooooooooooooooooooooooooooooooooooooooooooooooooong
-        == 1 ->
+        loooooooooooooooooooooooooooooooooooooooooooooooooooooooong| loooooooooooooooooooooooooooooooooooooooooooooooooooooooong
+        == 1  ->
```

Comment:
I wonder why this space moved?

Reply:
This output file shouldn't be checked in, and it's not updated consistently unless we remember to. So I'd guess the current checked in version is from some older version of the prettyprinter.